pkg/cmd/render/render.go

```diff
@@ -79,6 +81,7 @@ func (r *renderOpts) AddFlags(fs *pflag.FlagSet) {
 	fs.StringArrayVar(&r.etcdServerURLs, "manifest-etcd-server-urls", r.etcdServerURLs, "The etcd server URL, comma separated.")
 	fs.StringVar(&r.etcdServingCA, "manifest-etcd-serving-ca", r.etcdServingCA, "The etcd serving CA.")
 	fs.StringVar(&r.clusterConfigFile, "cluster-config-file", r.clusterConfigFile, "Openshift Cluster API Config file.")
+	fs.StringVar(&r.clusterAuthFile, "cluster-auth-file", r.clusterAuthFile, "Openshift Cluster Authentication API Config file.")
```

Comment:
in cluster kap-o users the authentication object to extract the issuers, and it made sense to keep the workflow same for render code in kap-o (process of getting the object is different but the contract is the same)

also if it was not kap-o render it will need to be done at the bootstrap-host by installer script, and that is something we are trying to reduce. The bootstrap-host's contract is a directory which has all the user and installer generated configuration, keep the abstraction easier to manage.